dagger.json

```diff
@@ -48,7 +48,7 @@
     }
   ],
   "source": ".dagger",
-  "engineVersion": "v0.12.7",
+  "engineVersion": "v0.12.6",
```

Comment:
Why this version got revert back to 0.12.6?

Reply:
(no idea how this happened btw)